src/screens/createWallet/SetPasswordScreen.tsx

```diff
@@ -101,17 +105,9 @@ function SetPasswordScreen(): JSX.Element {
                     const hasUpperCase = /[A-Z]/.test(val);
                     const hasLowerCase = /[a-z]/.test(val);
                     const hasNumbers = /\d/.test(val);
-                    const hasNonalphas = /\W/.test(val);
-                    if (
-                      !(
-                        hasUpperCase &&
-                        hasLowerCase &&
-                        hasNumbers &&
-                        hasNonalphas
-                      )
-                    ) {
+                    if (!(hasUpperCase && hasLowerCase && hasNumbers)) {
                       // eslint-disable-next-line max-len
-                      return 'Password should contain symbols in upper and lower cases, numbers, and special characters';
+                      return 'Password should contain symbols in upper and lower cases, numbers';
```

Comment:
```suggestion
                      return 'Password should contain numbers and symbols in upper and lower cases.';
```
because now it looks unfinished: 

<img width="411" alt="Screenshot 2024-10-21 at 19 06 42" src="https://github.com/user-attachments/assets/7155f9d0-8814-4a38-9950-129e9349ce55">